src/Sylius/Bundle/AdminApiBundle/Form/Type/AddressType.php

```diff
@@ -41,14 +41,31 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'required' => false,
                 'label' => 'sylius.form.checkout.addressing.different_billing_address',
             ])
+            ->add('differentShippingAddress', CheckboxType::class, [
+                'mapped' => false,
+                'required' => false,
+                'label' => 'sylius.form.checkout.addressing.different_shipping_address',
+            ])
             ->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $event) {
                 $orderData = $event->getData();
 
-                if (isset($orderData['shippingAddress']) && (!isset($orderData['differentBillingAddress']) || false === $orderData['differentBillingAddress'])) {
-                    $orderData['billingAddress'] = $orderData['shippingAddress'];
+                if (
+                    isset($orderData['billingAddress']) &&
+                    (!isset($orderData['differentBillingAddress']) || false === $orderData['differentBillingAddress']) &&
+                    (!isset($orderData['differentShippingAddress']) || false === $orderData['differentShippingAddress'])
+                ) {
+                    $orderData['shippingAddress'] = $orderData['billingAddress'];
+                }
 
-                    $event->setData($orderData);
+                if (
+                    isset($orderData['shippingAddress']) &&
+                    (!isset($orderData['differentBillingAddress']) || false === $orderData['differentBillingAddress']) &&
```

Comment:
Can we simplify this line?
```suggestion
                    (isset($orderData['differentBillingAddress']) && true === $orderData['differentBillingAddress']) &&
```
I would need explanation of it. Is it also possible to check both boxes at the same time?